### PR TITLE
Create Deanonimization.md

### PR DESCRIPTION
**Pull request by RiccardoMasutti**
New chapter about Bitcoin deanonimization and privacy best practices.

### CODE REVIEW

File: docs/using-wasabi/Deanonimization.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+{
+  "title": "How you can be de-anonymized using Bitcoin",
+  "description": "An explanation of the ways a Bitcoin transaction can be linked to someone's real identity. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
+}
+---
+
+# How you can be de-anonymized using Bitcoin
+
+Sadly, there are hundreds of ways a Bitcoin transaction can be linked to someone's real identity, even by using Wasabi. True pseudonymity against a resourceful adversary is very difficult to achieve.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
Sadly, there are many ways a Bitcoin transaction can be linked to someone's real identity, even by using Wasabi.
True pseudonymity against a resourceful adversary is very difficult to achieve.
```

File: docs/using-wasabi/Deanonimization.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Any sincere approach to anonymity in Bitcoin requires a holistic use of encryption and communication tools.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
Any real approach to anonymity in Bitcoin requires a total use of encryption and communication tools.
```

File: docs/using-wasabi/Deanonimization.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Over the course of just a few months, you could come into contact with hundreds of Bitcoin addresses.
+It is often only necessary to associate just one of these addresses with your real identity to work out your real identity.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
It is often only necessary to associate just one of these addresses to your real identity to de-anonymize you.
```

File: docs/using-wasabi/Deanonimization.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+
+[[toc]]
+
+## Revealing your Bitcoin address before it goes into the blockchain could let others track you
+
+As soon as a Bitcoin address is entered into the blockchain, it is publicly recorded in an immutable global database.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
As soon as a Bitcoin address is entered into the blockchain, it is publicly recorded in an immutable distributed ledger.
```

File: docs/using-wasabi/Deanonimization.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+Before that happens, it's likely that only two parties—the recipient and sender—had knowledge of this address.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
Before that happens, it's likely that only two parties (the recipient and the sender) had knowledge of this address.
```

File: docs/using-wasabi/Deanonimization.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+If you are making a search for an address that has not yet appeared on the blockchain, either you are the owner of this address, or someone is requesting a payment from you.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
If you are making a search for an address that is not in the blockchain, either you are the owner of this address, or someone is requesting a payment from you.
```

File: docs/using-wasabi/Deanonimization.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+To avoid being tracked in this way, it is important to make all payment requests and other mentions of addresses via encrypted channels:
+- Make sure the website you are visiting has HTTPS enabled when you search for Bitcoin addresses
+- Use VPNs and Tor when possible. You should check addresses with Blockstream Explora via their [.onion v3 address](http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion/)
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
- Use VPNs and Tor when possible. You should check addresses with Blockstream Explorer via their [.onion v3 address](http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion/)
```

File: docs/using-wasabi/Deanonimization.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+[[toc]]
+
+## Revealing your Bitcoin address before it goes into the blockchain could let others track you
+
+As soon as a Bitcoin address is entered into the blockchain, it is publicly recorded in an immutable global database.
+Before that happens, it's likely that only two parties—the recipient and sender—had knowledge of this address.
+If you are making a search for an address that has not yet appeared on the blockchain, either you are the owner of this address, or someone is requesting a payment from you.
+To avoid being tracked in this way, it is important to make all payment requests and other mentions of addresses via encrypted channels:
+- Make sure the website you are visiting has HTTPS enabled when you search for Bitcoin addresses
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
- Make sure the website you are visiting has HTTPS enabled when you search for Bitcoin addresses.
```

File: docs/using-wasabi/Deanonimization.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+- Use VPNs and Tor when possible. You should check addresses with Blockstream Explora via their [.onion v3 address](http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion/)
+- Encrypt your communications with PGP (emails, files and text) and OTR (chats)
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
- Encrypt your communications with PGP (emails, files and text) and OTR (chats).
```

File: docs/using-wasabi/Deanonimization.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+
+
+## Possessing a wallet file might be enough proof that you control Bitcoin
+
+A signed message is the only strong proof that you own a Bitcoin address without revealing the private key.
+Be careful when signing messages using your Bitcoin keys, though.
+Signed messages are a great feature, but they allow anybody to know that you control the Bitcoin address you signed from.
+If someone is trying to de-anonymize you, they might be satisfied with a weaker form of proof than a signed message: Knowing that you keep records of public addresses is enough evidence for someone to draw the conclusion that you are involved in Bitcoin transactions and the Bitcoin blockchain will point them exactly to which transactions.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
If someone is trying to de-anonymize you, they might be satisfied with a weaker form of proof than a signed message: knowing that you keep records of public addresses is enough evidence for someone to draw the conclusion that you are involved in Bitcoin transactions and the Bitcoin blockchain will point them exactly to which transactions.
```

File: docs/using-wasabi/Deanonimization.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+All they have to do is search for the address you recorded.
+Wasabi stores and encrypts public addresses and private keys.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]: I think this sentence is wrong. @lontivero ?

**Comment by yahiheb** [comment]
> You can safely store your wallet in an external USB drive, a cloud storage service, or even your email account if the private keys are sufficiently encrypted.

> To protect yourself, you should encrypt all backups with a second password or PGP.

I am not sure if this is recommended.

File: docs/using-wasabi/Deanonimization.md (hunk 11 above)
Comment (molnard) [reply docs/using-wasabi/Deanonimization.md]: IMO only the private key is encrypted.

Comment (yahiheb) [reply docs/using-wasabi/Deanonimization.md]: When you say `private key` do you mean the `Extended Master Private Key`?

File: docs/using-wasabi/Deanonimization.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+If someone is trying to de-anonymize you, they might be satisfied with a weaker form of proof than a signed message: knowing that you keep records of public addresses is enough evidence for someone to draw the conclusion that you are involved in Bitcoin transactions and the Bitcoin blockchain will point them exactly to which transactions.
+All they have to do is search for the address you recorded.
+Wasabi stores and encrypts public addresses and private keys.
+You can safely store your wallet in an external USB drive, a cloud storage service, or even your email account if the private keys are sufficiently encrypted.
+But anybody with access to these mediums could estimate what addresses you control and the transactions you make.
+To protect yourself, you should encrypt all backups with a second password or PGP.
+
+## Be careful with public spaces
+
+Unless your intention is to receive public donations or payments with full transparency, publishing a Bitcoin address on any public space such as a website or social network is not a good idea when it comes to privacy.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
Unless your intention is to receive public donations or payments with full transparency, publishing a Bitcoin address on any public space such as a website or a social network is not a good idea when it comes to privacy.
```

File: docs/using-wasabi/Deanonimization.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+If you choose to do so, always remember that if you move any funds with this address to one of your other addresses, they will be publicly tainted by the history of your public address.
+Additionally, you might also want to be careful not to publish information about your transactions and purchases that could allow someone to identify your Bitcoin addresses.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
Additionally, you might also want to be careful not to publish information about your transactions that could allow someone to identify your Bitcoin addresses.
```

File: docs/using-wasabi/Deanonimization.md (hunk 11 above)
Comment (lontivero) [reply docs/using-wasabi/Deanonimization.md]: Wasabi stores (and does not encrypt) the at the time generated public keys. Wasabi stores the encrypted master private key.

Comment (yahiheb) [reply docs/using-wasabi/Deanonimization.md]: is this batter? `Wasabi encrypts and stores the master private key.`

another question is the `Extended Master Private Key` the same as `master private key`?

File: docs/using-wasabi/Deanonimization.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+If someone is trying to de-anonymize you, they might be satisfied with a weaker form of proof than a signed message: knowing that you keep records of public addresses is enough evidence for someone to draw the conclusion that you are involved in Bitcoin transactions and the Bitcoin blockchain will point them exactly to which transactions.
+All they have to do is search for the address you recorded.
+Wasabi stores and encrypts public addresses and private keys.
+You can safely store your wallet in an external USB drive, a cloud storage service, or even your email account if the private keys are sufficiently encrypted.
+But anybody with access to these mediums could estimate what addresses you control and the transactions you make.
+To protect yourself, you should encrypt all backups with a second password or PGP.
+
+## Be careful with public spaces
+
+Unless your intention is to receive public donations or payments with full transparency, publishing a Bitcoin address on any public space such as a website or a social network is not a good idea when it comes to privacy.
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
Unless your intention is to receive public donations or payments with full transparency, publishing a Bitcoin address on any public space such as a website or a social network is not a good practice when it comes to privacy.
```

File: docs/using-wasabi/Deanonimization.md [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+[[toc]]
+
+## Revealing your Bitcoin address before it goes into the blockchain could let others track you
+
+As soon as a Bitcoin address is entered into the blockchain, it is publicly recorded in an immutable distributed ledger.
+Before that happens, it's likely that only two parties (the recipient and the sender) had knowledge of this address.
+If you are making a search for an address that is not in the blockchain, either you are the owner of this address, or someone is requesting a payment from you.
+To avoid being tracked in this way, it is important to make all payment requests and other mentions of addresses via encrypted channels:
+- Make sure the website you are visiting has HTTPS enabled when you search for Bitcoin addresses.
+- Use VPNs and Tor when possible. You should check addresses with Blockstream Explorer via their [.onion v3 address](http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion/).
```
Comment (yahiheb) [inline docs/using-wasabi/Deanonimization.md]:
```suggestion
- Use VPNs and Tor when possible.
You should check addresses with Blockstream Explorer via their [.onion v3 address](http://explorerzydxu5ecjrkwceayqybizmpjjznk5izmitf2modhcusuqlid.onion/).
```

File: docs/using-wasabi/Deanonimization.md (hunk 11 above)
Comment (lontivero) [reply docs/using-wasabi/Deanonimization.md]: > another question is the Extended Master Private Key the same as master private key?

No. The `Extended Master Private Key` is the combination of the (`Master Private Key` + `Chain code`) where the  `Master Private Key` is simple a private key. Wasabi encrypts the `Master Private Key` and saves the `Chain code` unencrypted.

**Pull request merged**